### PR TITLE
remove scrollbar from external resource widget config

### PR DESCRIPTION
**Pull request by 3nids**
following discussion in #44897
ping @DelazJ 

Screen Recording 2021-09-01 at 09 48 50

### CODE REVIEW

**Comment by 3nids** [comment]
I think this is much better like this

**Comment by DelazJ** [comment]
You beat me on the final line 😄 
And I'd say: "It's much much better!" 👍 Thanks.

If I may, the next round could now be to see whether we can merge some of the groups, and avoid embedded subgroups. To make it look more coherent, instead of (I may be wrong as I'm not a user - sorry!) an addition of features.

File: src/ui/editorwidgets/qgsexternalresourceconfigdlg.ui [hunk 1]
```diff
@@ -632,7 +571,7 @@
  <resources/>
  <connections/>
  <buttongroups>
-  <buttongroup name="mStorageButtonGroup"/>
   <buttongroup name="mRelativeButtonGroup"/>
+  <buttongroup name="mStorageButtonGroup"/>
```
Comment (DelazJ) [inline src/ui/editorwidgets/qgsexternalresourceconfigdlg.ui:575]: Do you know what these refer to? I couldn't find the widgets in my Qt Designer (or I was too tired?)

**Comment by troopa81** [comment]
Nice! :+1:

**Review by nyalldawson — APPROVED** [review]

**Pull request merged**